### PR TITLE
Implemented extended patient details on EHR side. Minor fixes.

### PR DESCRIPTION
**Pull request by RuslanDidyk**

### CODE REVIEW

File: ehr/app/src/main/java/org/hl7/gravity/refimpl/sdohexchange/dto/converter/PatientToDtoConverter.java [hunk 1]
```diff
@@ -3,49 +3,105 @@
...
+        .orElse(NOT_AVAILABLE));
 
+    List<ContactPoint> telecom = patient.getTelecom();
+    //TODO: Reimplement after clarification
+    //Get phone number
+    patientDto.setPhone(telecom.stream()
+        .filter(t -> ContactPoint.ContactPointSystem.PHONE.equals(t.getSystem()))
+        .map(ContactPoint::getValue)
+        .findFirst()
+        .orElse(NOT_AVAILABLE));
```
Comment (yuflyud) [inline ehr/app/src/main/java/org/hl7/gravity/refimpl/sdohexchange/dto/converter/PatientToDtoConverter.java:66]: Why not just return null and UI will will decide what to show, N/A or something else?

File: ehr/app/src/main/java/org/hl7/gravity/refimpl/sdohexchange/dto/converter/TaskBundleToDtoConverter.java [hunk 2]
```diff
@@ -44,25 +44,26 @@
         .collect(Collectors.toList());
   }
 
-  protected TaskDto composeTaskDto(Task t, Map<String, ServiceRequest> srMap, Map<String, Organization> orgMap) {
-    TaskDto taskDto = new TaskDto(t.getIdElement()
+  protected TaskDto composeTaskDto(Task tasl, Map<String, ServiceRequest> srMap, Map<String, Organization> orgMap) {
```
Comment (yuflyud) [inline ehr/app/src/main/java/org/hl7/gravity/refimpl/sdohexchange/dto/converter/TaskBundleToDtoConverter.java:47]: task not tasl

File: ehr/app/src/main/java/org/hl7/gravity/refimpl/sdohexchange/service/CbroTaskCreateService.java [hunk 3]
```diff
@@ -32,11 +32,12 @@
 
   public void createTask(IGenericClient cbroClient, Task task) throws CbroTaskCreateException {
     Task t = externalizeResource(task.copy());
+    //TODO: Remove this after Logica bug response
+    t.setMeta(null);
```
Comment (yuflyud) [inline ehr/app/src/main/java/org/hl7/gravity/refimpl/sdohexchange/service/CbroTaskCreateService.java:36]: Why do we need this? the original task does not have a profile filed set anyway.

Comment (RuslanDidyk) [reply ehr/app/src/main/java/org/hl7/gravity/refimpl/sdohexchange/service/CbroTaskCreateService.java:36]: Yes, it has. TaskBundleFactory.143 line

File: ehr/app/src/main/java/org/hl7/gravity/refimpl/sdohexchange/dto/converter/PatientToDtoConverter.java (hunk 1 above)
Comment (RuslanDidyk) [reply ehr/app/src/main/java/org/hl7/gravity/refimpl/sdohexchange/dto/converter/PatientToDtoConverter.java:66]: Okay, will fix this.

Comment (RuslanDidyk) [reply ehr/app/src/main/java/org/hl7/gravity/refimpl/sdohexchange/dto/converter/PatientToDtoConverter.java]: Fixed.

**Pull request merged**